src/common/libsubprocess/local.c

```diff
@@ -492,7 +492,7 @@ static int local_child (flux_subprocess_t *p)
     // Change working directory
     if ((cwd = flux_cmd_getcwd (p->cmd)) && chdir (cwd) < 0) {
         fprintf (stderr,
-                 "Couldn't change dir to %s: going to /tmp instead: %s\n",
+                 "Could not change dir to %s: going to /tmp instead: %s\n",
```

Comment:
minor suggestion, strerror seems like it should go before "going to /tmp instead", e.g.
```
Could not change dir to /foo: No such file or directory. Going to /tmp instead.
```